Maddori.Apple/Maddori.Apple/Screen/Home/AddFeedback/AddFeedbackContentViewController.swift

```diff
@@ -0,0 +1,276 @@
+    
+    override func setupNavigationBar() {
+        super.setupNavigationBar()
+        
+        let button = removeBarButtonItemOffset(with: backButton, offsetX: 10)
+        let backButton = makeBarButtonItem(with: button)
+        let closeButton = makeBarButtonItem(with: closeButton)
+        
+        navigationController?.navigationBar.prefersLargeTitles = false
+        navigationItem.largeTitleDisplayMode = .never
+        navigationItem.leftBarButtonItem = backButton
+        navigationItem.rightBarButtonItem = closeButton
+    }
+    
+    private func setupNotificationCenter() {
+        NotificationCenter.default.addObserver(self, selector: #selector(willShowKeyboard), name: UIResponder.keyboardWillShowNotification, object: nil)
+        NotificationCenter.default.addObserver(self, selector: #selector(willHideKeyboard), name: UIResponder.keyboardWillHideNotification, object: nil)
+    }
+    
+    private func setupDelegate() {
+        feedbackContentTextView.delegate = self
+    }
+    
+    private func didTappedBackButton() {
+        navigationController?.popViewController(animated: true)
+    }
+    
+    func didTappedCloseButton() {
+        self.dismiss(animated: true)
+    }
+    
+    func didTappedDoneButton() {
+        currentStepString = feedbackContentTextView.text
+        
+        if contentString.isEmpty {
+            contentString = currentStepString
+        } else {
+            contentString = contentString + "\n\n" + currentStepString
+        }
+        
+        NotificationCenter.default.removeObserver(self)
+        DispatchQueue.main.async {
+            switch self.step {
+            case .second:
+                self.navigationController?.pushViewController(AddFeedbackContentViewController(step: Step.third, content: self.contentString), animated: true)
+            case .third:
+                self.navigationController?.pushViewController(AddFeedbackContentViewController(step: Step.fourth, content: self.contentString), animated: true)
+            case .fourth:
+                print(self.contentString)
+                // FIXME: 키워드 작성하는 마지막 단계 VC가 생기면 그 VC로 연결
+            }
+        }
+        // FIXME: currentStepString과 쌓여가는 contentString 볼 수 있도록 해 둔 코드 -> 머지 전에 지우기
+        print("currentStepString: \(currentStepString)")
+        print("contentString: \(contentString)")
+    }
+    
+    // MARK: - selector
+    
+    @objc private func willShowKeyboard(notification: NSNotification) {
+        guard let keyboardSize = (notification.userInfo?[UIResponder.keyboardFrameEndUserInfoKey] as? NSValue)?.cgRectValue else { return }
+        
+        feedbackContentTextView.snp.updateConstraints {
+            $0.bottom.equalTo(doneButton.snp.top).offset(-keyboardSize.height + 15)
+        }
+        UIView.animate(withDuration: 0.2, animations: {
+            self.doneButton.transform = CGAffineTransform(translationX: 0, y: -keyboardSize.height + 25)
+            self.view.layoutIfNeeded()
```

Comment:
p2;
다른 뷰와는 다르게 이 뷰들은 처음부터 키보드를 올려두고 시작하는 거라서 그런지 다음 페이지로 navigate할 때 뒤늦게 버튼이 올라오는 게 좀 어색하다는 느낌이 드는 것 같아요! 그래서 duration은 0으로 바꾸는 건 어떨까요? 아래 녹화화면 첨부했습니다!

```suggestion
        UIView.animate(withDuration: 0, animations: {
            self.doneButton.transform = CGAffineTransform(translationX: 0, y: -keyboardSize.height + 25)
            self.view.layoutIfNeeded()
```

https://user-images.githubusercontent.com/81340603/205493671-30ce3639-07de-4207-8988-9113f837dbbd.mp4
- 기존 화면

https://user-images.githubusercontent.com/81340603/205493668-dfb3e364-7cbc-4280-8c52-4f9dea60e908.mp4
- duration을 없앴을 경우

Reply:
와우... 훨씬 자연스럽네요 !

Reply:
[28604a7](https://github.com/DeveloperAcademy-POSTECH/MacC-Team-Maddori.Apple/pull/211/commits/28604a7b47746268fb45df11a52b7ce588d6c8a1)
와 이거 진짜 진저 인정 대박
훨 자연스럽습니당